.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Bug Report
+about: Report a problem with qBittorrent to help us resolve it.
+title: ''
+labels: ''
+assignees: ''
+---
+
+# Bug report
+
+<!--
+###############################################################################
+  WARNING!
+  IGNORING THE FOLLOWING TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED
+  AS INCOMPLETE/INVALID
+###############################################################################
+-->
+
+## Checklist
+
+<!--
+################################## IMPORTANT ##################################
+
+As you read and fulfill each of the following requirements below,
+put an "x" (without the quotes) between the square brackets.
+
+When doing so, ensure that no space is left between the square brackets and the "x", like so: [x]
+-->
+
+- [ ] I have read the **issue reporting guidelines** in the [contributing guidelines](https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md), so I know how to submit a good bug report with the required information
+- [ ] I have verified that **the issue is not fixed and is reproducible** in the **[latest version](https://www.qbittorrent.org/download.php)**
+- [ ] I have **searched** the issue tracker for similar bug reports including closed ones **to avoid posting a duplicate**
+- [ ] This is **not a support issue or question**, or the [wiki](https://github.com/qbittorrent/qBittorrent/wiki) did not contain the required information and the [forum](https://qbforums.shiki.hu/) and [subreddit](https://www.reddit.com/r/qBittorrent/) were not able to assist me, so I am asking here
+- [ ] I have **pasted/attached the settings file and relevant log(s)** (at least the most recent one) in the **Attachments** section at the bottom of the report and filled in all the other required information below
+
+## Description
+
+<!--
+################################## IMPORTANT ##################################
+
+Delete each "(type here)" indicator and type your text in their place in the subsections below.
+
+Please make sure the description is worded well enough to be understood.
+Provide steps to reproduce the issue, any additional relevant information, suggested solution (if applicable) and as much context and examples as possible.
+For more information consult the Contributing Guidelines at https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md.
+
+Do not forget about the mandatory attachments!
+
+Use the Preview tab before posting to make sure your report looks like it is formatted properly.
+-->
+
+### qBittorrent version, install method (installer, compiled from git, ...), and operating system version
+
+(type here)
+
+### If on Linux, `libtorrent-rasterbar` and `Qt` version
+
+(type here)
+
+### What is the problem
+
+(type here)
+
+### What is the expected behavior
+
+(type here)
+
+### Detailed steps to reproduce the problem
```

Comment:
I would swap positions with
 "What is the expected behavior"

Reply:
resolved.